.github/workflows/maven-verify.yml

```diff
@@ -26,6 +26,5 @@ jobs:
     name: Verify
     uses: apache/maven-gh-actions-shared/.github/workflows/maven-verify.yml@v4
     with:
-      ff-maven: "4.0.0-beta-5"                     # Maven version for fail-fast-build
-      maven-matrix: '[ "4.0.0-beta-5" ]'
-      jdk-matrix: '[ "17", "21" ]'
+      maven4-build: true
```

Comment:
What's the effect of this flag ?

Reply:
https://github.com/apache/maven-gh-actions-shared?tab=readme-ov-file#build-with-maven-4-only